### PR TITLE
github-management: update README.md

### PR DESCRIPTION
**Pull request by wangxy518**
responsibilites->responsibilities

### CODE REVIEW

**Comment by mrbobbytables** [comment]
the document being linked to is called the current spelling mistake. Can you rename the file as well so it doesn't 404 with this change?

File: github-management/README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ and organizations. We have also created a number of tools to automate setup and
 enforcement of these policies.
 
 These polices are overseen by the
-[GitHub Management subproject](subproject-responsibilites.md) of the Contributor
```
Comment (fejta) [inline github-management/README.md:13]: This is the correct link

File: github-management/README.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ and organizations. We have also created a number of tools to automate setup and
...
+[GitHub Management subproject](subproject-responsibilities.md) of the Contributor
```
Comment (fejta) [inline github-management/README.md:13]: This is a 404

**Comment by mrbobbytables** [comment]
superceded by https://github.com/kubernetes/community/pull/4993
/close

**Pull request closed without merging**